Makefile

```diff
@@ -155,31 +155,16 @@ podman-build: test ## Build container image with the manager.
 docker-push: ## Push docker image with the manager.
 	docker push ${IMG}
 
-# PLATFORMS defines the target platforms for  the manager image be build to provide support to multiple
```

Comment:
This was added following the operator SDK instructions, can we leave for the future upgrades of the sdk ?

Reply:
will set back.
Just did not want it to be confusing